build/config-builder.js

```diff
@@ -42,6 +42,9 @@ const BASE_CONFIG = {
 
   // Whether or not the build is packaged in an electron distributable
   packaged: false,
+
+  // The Google Analytics tracking identifier to use when instrumenting.
+  google_analytics_UA: 'UA-76122102-1'
```

Comment:
Don't we want this to be secret-ish? At least coming from somewhere outside of the tree means every fork won't report to our account.